ext/napi/function.rs

```diff
@@ -74,14 +75,15 @@ pub fn create_function<'a>(
   function
 }
 
-#[allow(clippy::not_unsafe_ptr_arg_deref)]
-pub fn create_function_template<'a>(
+/// # Safety
+/// env_ptr must be valid
+pub unsafe fn create_function_template<'a>(
   env_ptr: *mut Env,
```

Comment:
I think this can now be `&mut Env`?

Reply:
`&mut Env` is annoying with the lifetime of the created js object.